doc/changelog_update.md

```diff
@@ -0,0 +1,59 @@
+# Changelog Update
+
+If you want to help with updating the [changelog][changelog], you're in the right place.
+
+## When to update
+
+The changelog is ideally updated during the week before an upcoming stable
+release. Typos and other small fixes/additions are always welcome. You can find
```

Comment:
`Typos and other small fixes/additions are always welcome.` should be at the beginning of this paragraph and emphasized to encourage potential typo fixes.